Cargo.toml

```
@@ -29,7 +29,7 @@ indexmap = { version = "2.0.0", features = ["serde"] }
 itertools = "0.11.0"
 log = "0.4.20"
 pin-project-lite = "0.2.13"
-reqwest = { version = "0.11.20", features = ["blocking", "native-tls-vendored", "stream"] }
+reqwest = { version = "0.11.20", features = ["blocking", "stream"] }
```

```suggestion
reqwest = { version = "0.11.20", features = ["blocking", "stream"], default-features = false }
```

default-features = falseがないとopensslのリンクが残りますね